### PR TITLE
Update ENSO notebook

### PR DESCRIPTION
**Pull request by acordonez**

### CODE REVIEW

**Comment by lee1043** [comment]
@acordonez It looks like downloading is keep failing on my end, could you please check how it work on your end?

```
Downloading: 'misc_demo_data/fx/sftlf.GPCP-IP.1x1.nc' from 'https://pcmdiweb.llnl.gov/pss/pmpdata/' in: demo_data/misc_demo_data/fx/sftlf.GPCP-IP.1x1.nc
Download failed
```

**Comment by lee1043** [comment]
Below is what I get if I run the `cdat_info.download_sample_data_files` without `try`. @acordonez @gleckler1 any idea?

```
----> 5     cdat_info.download_sample_data_files("data_files.txt", demo_data_directory)
      6     print("All files downloaded")
      7 #except:

/opt/anaconda3/envs/pcmdi_metrics_dev/lib/python3.9/site-packages/cdat_info-8.2.1-py3.9.egg/cdat_info/cdat_info_src.py in download_sample_data_files(files_md5, path)
    357         n0 = 1
    358     for sample in samples[n0:]:
--> 359         good_md5, name = sample.split()
    360         local_filename = os.path.join(path, name)
    361         try:
```

**Comment by acordonez** [comment]
@lee1043 I'm seeing "enso_data_files.txt" instead of "data_files.txt" in my version of the notebook. Is that a change that I didn't pick up?

**Comment by acordonez** [comment]
@lee1043 @gleckler1 I checked the data_files.txt file from the Demo 0 notebook and it has an empty line at the end, which would cause a failure message.

**Comment by lee1043** [comment]
@acordonez I copied the error message from a different notebook (Demo 0), sorry for the confusion!

**Review by lee1043 — APPROVED** [review]

**Comment by lee1043** [comment]
I confirmed the ENSO demo working fine. I will fix the Demo 0 issue in separate, perhaps I need to fix it from the web server.

**Comment by lee1043** [comment]
I think I have fixed the `data_files.txt` issue (empty line at the end) earlier in the web server, but it looks like somehow reverted. I will investigate on that.

File: doc/jupyter/Demo/Demo_6_ENSO.ipynb [hunk 1]
```diff
@@ -74,9 +66,9 @@
    "outputs": [],
    "source": [
     "# This is where you will be downloading the sample_data\n",
-    "demo_data_directory = \"demo_data_enso_new_obs\"\n",
+    "demo_data_directory = \"demo_data\"\n",
```
[the remark this answers is not in the document]
Comment (acordonez) [reply doc/jupyter/Demo/Demo_6_ENSO.ipynb:69]: Screen Shot 2022-04-05 at 5 00 52 PM
@lee1043 confirming that notebook works with sample data downloaded to "demo_data" directory

**Comment by acordonez** [comment]
@lee1043 If we can't fix the issue on the server, we can do some clean up to remove blank lines after download in the Demo 0 notebook.

File: doc/jupyter/Demo/Demo_6_ENSO.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (lee1043) [reply doc/jupyter/Demo/Demo_6_ENSO.ipynb:69]: @acordonez thanks for checking!

**Pull request merged**